lib/annotate_rb/parser.rb

```diff
@@ -127,6 +127,15 @@ def add_wrapper_options_to_parser(option_parser)
         @options[:wrapper_open] = wrapper_open
       end
 
+      option_parser.on("--classes-default-to-s class",
```

Comment:
Could we move this block outside of `#add_wrapper_options_to_parser`? Perhaps into `#add_model_options_to_parser` instead?

Reply:
Done 👍🏻